core/plugins/org.fusesource.ide.camel.validation/src/org/fusesource/ide/camel/validation/diagram/BasicNodeValidator.java

```diff
@@ -230,8 +209,7 @@ private IMarker createOrReuseMarker(IResource resource, AbstractCamelModelElemen
 	 * @throws CoreException
 	 */
 	private IMarker createMarker(IResource resource, final String message, final int severity, Map<String, Object> attributesForPosition) throws CoreException {
-		IMarker res;
-		res = resource.createMarker(IFuseMarker.MARKER_TYPE);
+		IMarker res = resource.createMarker(IFuseMarker.MARKER_TYPE);
```

Comment:
Can we change some of these local variables to use a different name than "res" which confuses it with the IResource a bit?